.github/scripts/pre-commit.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -e
+
+checkSecretFilesForPlainText() {
+  echo 'Verifying secret files are not in plaintext...'
+  set +e
+  # read list of files containing secrets
+  IFS=$'\n' read -d '' -r -a secrets < .github/resources/.sensitive-files
+
+  # make list of files to be committed by printing out a newline
+  # separated list of files staged for commit to a temporary file
+  # and reading it as an array
+  tmpfile=$(mktemp)
+  git diff --cached --name-only --diff-filter=ACM > $tmpfile
+  IFS=$'\n' read -d '' -r -a commits < $tmpfile
+
+  # for all files to be committed which are a secret file,
+  # grep for the header "$ANSIBLE_VAULT;1.1;AES256"
+  # if it's not there, abort.
```

Comment:
This is really nitpicky, but the commend on lines 10-12 doesnt have a period at the end.

Reply:
comment*

Reply:
True, just copied it from the pom, but I'll fix this.